### ISSUE TITLE
ESMFMKFILE does not seem to be properly set for Windows environments

### ISSUE THREAD

**Issue by Zeitsperre** [conda-forge/esmf-feedstock#105]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

It appears that `ESMFMKFILE` is not being properly set for packages build upon ESMF in Windows. This env var should probably be set within the conda-forge package.

```
(base) %SRC_DIR%>conda activate "%PREFIX%" 
import: 'xscen'
Traceback (most recent call last):
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\esmpy\interface\loadESMF.py", line 26, in <module>
    esmfmk = os.environ["ESMFMKFILE"]
             ~~~~~~~~~~^^^^^^^^^^^^^^
  File "<frozen os>", line 679, in __getitem__
KeyError: 'ESMFMKFILE'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\xesmf\backend.py", line 22, in <module>
    import esmpy as ESMF
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\esmpy\__init__.py", line 112, in <module>
    from esmpy.api.esmpymanager import *
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\esmpy\api\esmpymanager.py", line 9, in <module>
    from esmpy.interface.cbindings import *
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\esmpy\interface\cbindings.py", line 13, in <module>
    from esmpy.interface.loadESMF import _ESMF
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\esmpy\interface\loadESMF.py", line 28, in <module>
    raise ImportError('The ESMFMKFILE environment variable is not available.')
ImportError: The ESMFMKFILE environment variable is not available.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\bld\xscen_1692289661119\test_tmp\run_test.py", line 2, in <module>
    import xscen
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\xscen\__init__.py", line 5, in <module>
    from . import (
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\xscen\aggregate.py", line 16, in <module>
    import xesmf as xe
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\xesmf\__init__.py", line 4, in <module>
    from .frontend import Regridder, SpatialAverager
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\xesmf\frontend.py", line 12, in <module>
    from .backend import Grid, LocStream, Mesh, add_corner, esmf_regrid_build, esmf_regrid_finalize
  File "C:\bld\xscen_1692289661119\_test_env\Lib\site-packages\xesmf\backend.py", line 24, in <module>
    import ESMF
ModuleNotFoundError: No module named 'ESMF'
```

### Installed packages

```shell
Prefix: C:\Miniforge

  Updating specs:

   - xscen=0.6.0[build=pyhd8ed1ab_0]
   - cartopy
   - cf_xarray[version='>=0.7.6']
   - cftime
   - clisops[version='>=0.9.5']
   - dask-core
   - flox
   - fsspec
   - geopandas
   - h5netcdf
   - intake
   - intake-esm[version='>=2022.9.18']
   - matplotlib-base
   - netcdf4
   - numpy
   - pandas !=1.5.3
   - pyarrow
   - pygeos
   - python[version='>=3.8']
   - pyyaml
   - rechunker
   - shapely
   - xarray[version='<2023.3.0']
   - xclim[version='>=0.37']
   - xesmf[version='>=0.7']
   - zarr
   - pip
   - python
```

### Environment info

```shell
N/A (Running on Azure/conda-forge)
```

**Comment by Zeitsperre** [comment]
See also: https://github.com/conda-forge/staged-recipes/pull/23701

**Comment by aulemahal** [comment]
Hopefully, this will be fixed by #104!

**Comment by ocefpaf** [comment]
> I'm guessing this issue could maybe be transfered to the `esmpy-feedstock` repo.

`esmpy` is noarch, so it would be best if that setting is transferred to ESMF instead.

**Comment by xylar** [comment]
We should take care of this before #104 goes in.  But I don't have time to fix it until maybe later this week or over the weekend.

**Comment by zmoon** [comment]
Just wanted to note that this is not fixed in the versions currently available for Windows (8.4.0, 8.4.1, and 8.4.2, as far as I can tell).

Setting it manually seems to work though. For example, in PowerShell:
```powershell
$env:ESMFMKFILE = "<conda env dir>\Library\lib\esmf.mk"
```
